newIDE/app/src/SceneEditor/index.js

```diff
@@ -1194,6 +1194,26 @@ export default class SceneEditor extends React.Component<Props, State> {
             resourceSources={resourceSources}
             resourceExternalEditors={resourceExternalEditors}
             onChooseResource={onChooseResource}
+            onComputeAllVariableNames={() => {
+              const variablesEditedInstance = this.state
+                .variablesEditedInstance;
+              if (!variablesEditedInstance) {
+                return [];
+              }
+              const variablesEditedObject = getObjectByName(
+                project,
+                layout,
+                variablesEditedInstance.getObjectName()
+              );
+              return variablesEditedObject
+                ? EventsRootVariablesFinder.findAllObjectVariables(
+                    project.getCurrentPlatform(),
+                    project,
+                    layout,
+                    variablesEditedObject
+                  )
+                : [];
```

Comment:
This seems wrong, you check the object of the edited instance, but we're in the object editor, so we're editing an object.

instead, just pass `this.state.editedObjectWithContext.object` to the last argument of findAllObjectVariables?